site/docs/install-centos.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: documentation
+title: Installing Bazel on CentOS
+---
+
+# <a name="centos"></a>Install Bazel on CentOS
+
+Supported CentOS Linux platforms:
+
+*   CentOS 7.3
+*   CentOS 7.4
+
+Install Bazel on CentOS using one of the following methods:
+
+*   [Use our custom Bazel COPR repository (experimental)](#install-on-centos)
+*   [Compile Bazel from source (experimental)](install-compile-source.md)
+
+Bazel comes with two completion scripts. After installing Bazel, you can:
+
+*   access the [bash completion script](install.md)
```

Comment:
Capitalize the first word of each bullet.

Reply:
@spomorski thanks for the review.  Since @vbatts has built a COPR for bazel 0.5.3 in https://github.com/bazelbuild/bazel/pull/3721, once that merges I'll abandon this pull request.

Reply:
#3721 has been merged, what would you like to do with this PR?

Reply:
@katre since #3721 merged, I'll close this.  Thanks @vbatts for integrating the changes in this review and bazel 0.5.3.

Reply:
@katre thanks for providing an initial review.  Cheers!